sdcm/tester.py

```diff
@@ -1473,8 +1473,8 @@ def get_data_set_size(self, cs_cmd):
     @retrying(n=60, sleep_time=60, allowed_exceptions=(AssertionError,))
     def wait_data_dir_reaching(self, size, node):
         query = '(sum(node_filesystem_size{{mountpoint="{0.scylla_dir}", ' \
-            'instance=~"{1.private_ip_address}"}})-sum(node_filesystem_avail{{mountpoint="{0.scylla_dir}", ' \
```

Comment:
this change is maybe correct, but again the not relevant to running from local machine.
might be relevant to multi-dc cases

Reply:
> Those changes to node_setup would break AWS

You are right